app/assets/javascripts/components/emails/emails_card.es6.jsx

```diff
@@ -16,10 +16,13 @@ class EmailsCard extends Component {
   render() {
     var email = this.props.email;
     return (
-      <div style={StyleConstants.cards.show(this.props.media)}>
-        <a href={RouteConstants.emails.show(email.id)}>{`${email.sender} - ${email.subject}`}</a>
```

Comment:
@quinnromanek, whenever we want to render "clickable" components, we make use of our custom 'Clickable' component!